WeScan/Scan/ScannerViewController.swift

```diff
@@ -54,8 +57,8 @@ final class ScannerViewController: UIViewController {
         return toolbar
     }()
     
-    lazy private var autoScanButton: UIBarButtonItem = {
-        return UIBarButtonItem(title: NSLocalizedString("wescan.scanning.auto", tableName: nil, bundle: Bundle(for: ScannerViewController.self), value: "Auto", comment: "The auto button state"), style: .plain, target: self, action: #selector(toggleAutoScan))
+    lazy private var autoModeButton: UIBarButtonItem = {
```

Comment:
`autoMode` sounds pretty vague to me, especially without documentation. Can we make this property more descriptive and can we add some documentation?

Reply:
Hey, autoMode is documented on the `isAutoModeEnabled` property of the `CaptureSession` 😃